sandbox/src/components/DecisioningEngineDemo/DecisioningEngine.js

```diff
@@ -0,0 +1,111 @@
+import React, { useState, useEffect } from "react";
+import ContentSecurityPolicy from "../ContentSecurityPolicy";
+import "./DecisionEngineStyle.css";
+
+export default function DecisionEngine() {
+  const [realResponse, setRealResponse] = useState(null);
+
+  async function transformPayload(json) {
```

Comment:
Please merge `consequenceAdapter` into your branch (if you haven't already).  And then you can remove this `transformPayload` method.  It's taken care of elsewhere.